introduction.md

```diff
@@ -1,3 +1,10 @@
 Hi, my name is Carlos Eduardo, but you can call me Cadu.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?